src/screens/FormScreen.js

```diff
@@ -59,60 +68,69 @@ export const FormScreen = () => {
   // TODO: texts are hardcoded because they will come from the API later somewhen
   return (
     <SafeAreaViewFlex>
-      <ScrollView>
-        <WrapperWithOrientation orientation={orientation} dimensions={dimensions}>
-          <Mutation mutation={createQuery(QUERY_TYPES.APP_USER_CONTENT)}>
-            {(createAppUserContent) => (
-              <View style={{ padding: normalize(14) }}>
-                <BoldText>Name</BoldText>
-                <TextInput
-                  onChangeText={(text) => {
-                    setName(text);
-                  }}
-                  value={name}
-                  style={styles.inputField}
-                />
-                <BoldText>E-Mail</BoldText>
-                <TextInput
-                  onChangeText={(text) => {
-                    setEmail(text);
-                  }}
-                  value={email}
-                  style={styles.inputField}
-                  keyboardType="email-address"
-                />
-                <BoldText>Telefon</BoldText>
-                <TextInput
-                  onChangeText={(text) => {
-                    setPhone(text);
-                  }}
-                  value={phone}
-                  style={styles.inputField}
-                />
-                <BoldText>Ihre Mitteilung</BoldText>
-                <TextInput
-                  onChangeText={(text) => {
-                    setMessage(text);
-                  }}
-                  value={message}
-                  style={styles.textArea}
-                  multiline
-                  textAlignVertical="top"
-                />
-                <CheckBox
-                  checked={consent}
-                  onPress={() => setConsent(!consent)}
-                  title="Ich bin mit dem Speichern meiner Daten einverstanden?"
-                  checkedColor={colors.accent}
-                  containerStyle={styles.checkboxContainerStyle}
-                  textStyle={styles.checkboxTextStyle}
-                />
-                <Button title="Senden" onPress={() => submitForm(createAppUserContent)}></Button>
-              </View>
-            )}
-          </Mutation>
-        </WrapperWithOrientation>
-      </ScrollView>
+      <KeyboardAvoidingView
+        enabled={device.platform === 'ios'}
+        behavior={device.platform === 'ios' && 'padding'}
+        keyboardVerticalOffset={
+          device.platform === 'ios' && getHeaderHeight(orientation) + statusBarHeight(orientation)
```

Comment:
How and why getHeaderHeight and statusBarHeight can effect the Keyboard ?

Reply:
good question. i figured out, that the `KeyboardAvoidingView` has this vertical offset. the [default value is 0](https://reactnative.dev/docs/keyboardavoidingview/#keyboardverticaloffset).
our header affects the available space for the keyboard. you can see a wrong offset with a white area like padding above the keyboard.
so we need the exact size of header and tell it the `KeyboardAvoidingView`. we get the size with the new helper methods.
for iOS we have the special case, that the header is different in portrait and landscape. on landscape, it is smaller. so we need this additional case to be handled, which happens here with all the informations of device and orientation.

Reply:
brilliant !